### PR TITLE
NOSTORY: first efs commit

### PR DESCRIPTION
**Pull request by patrickmukumbu**
Hey @JohnNMwangi  please review and test this module for me to merge

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 3.74.3 |
```
Comment (j0hnmwangi) [inline examples/README.md:12]: Since the module works well with versions 3.x and higher including 4.x, do we need to lock it with 3.x?

**Review by j0hnmwangi — CHANGES_REQUESTED** [review]
Tested the module and works fine. See my comment on aws version.

File: examples/README.md (hunk 1 above)
Comment (patrickmukumbu) [reply examples/README.md:12]: It is not explicitly locked because it accepts any version above 3.5 @JohnNMwangi

**Review by j0hnmwangi — APPROVED** [review]
LGTM

**Pull request merged**